Build/CMakeModules/FindSQLite3.cmake

```diff
@@ -0,0 +1,46 @@
+# - Find sqlite3
```

Comment:
I wonder if it's not better to just add the sqlite source directly in the repo?

Reply:
That's common, but the sqlite parts won't get updated from upstream frequently (if at all).